keeper/keeper.go

```diff
@@ -135,10 +135,13 @@ func (k *Keeper) SendRestrictionFn(ctx context.Context, _, toAddr sdk.AccAddress
 	}
 
 	mintingDenom := k.ftfKeeper.GetMintingDenom(ctx).Denom
-	if coins.AmountOf(mintingDenom).IsPositive() {
-		err = k.PendingTransfers.Set(ctx, account.Address, *account)
+	if len(coins) != 1 || !coins.AmountOf(mintingDenom).IsPositive() {
+		return toAddr, fmt.Errorf("autocctp accounts can only receive %s coins", mintingDenom)
```

Comment:
For any future reviewers, just want to comment that this is where the actual change is made!